pkgs/applications/editors/vscode/default.nix

```diff
@@ -22,6 +22,7 @@ let
   rpath = lib.concatStringsSep ":" [
     atomEnv.libPath
     "${lib.makeLibraryPath [gtk2]}"
+    "${lib.makeLibraryPath [at_spi2_atk]}"
     "${lib.makeLibraryPath [libsecret]}/libsecret-1.so.0"
     "${lib.makeLibraryPath [libXScrnSaver]}/libXss.so.1"
     "${lib.makeLibraryPath [libxkbfile]}/libxkbfile.so.1"
```

Comment:
I wonder why there are absolute paths to files here. the rpath usually contain directories.